Anasayfa.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="Homework" content="task">
+    <title>Mert</title>
+</head>
+<body style>
+    <h1 style=>HOŞ GELDİNİZ!</h3>
+        <p>
+            <a href="Anasayfa.html" style="font-size: 20px;">

```

Comment:
```suggestion
            <a href="anasayfa.html" style="font-size: 20px;">

```
Yazılan etiketler doğru ancak best practise olarak sayfa linkleri küçük harfle yazılmalıdır.